modules/cram/templates/concat_vcf.sh

```diff
@@ -7,21 +7,39 @@ concat () {
   args+=("--allow-overlaps")
   args+=("--remove-duplicates")
   args+=("--output-type" "z")
-  args+=("--output" "!{vcfOut}")
+  args+=("--output" "unsorted_!{vcfOut}")
   args+=("--no-version")
   args+=("--threads" "!{task.cpus}")
-  args+=(!{vcfs})
+  for vcf in !{vcfs}
+  do
+    args+=("sorted_${vcf}")
+  done
 
   ${CMD_BCFTOOLS} "${args[@]}"
 }
 
+bcftools_sort () {
+  ${CMD_BCFTOOLS} sort -O z "unsorted_!{vcfOut}" --output "!{vcfOut}"
```

Comment:
use long opts instead of short for consistency/clarify: `--output-type` instead of `-O`